src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -333,12 +333,12 @@ private void loadTownAndNationLevels() {
 	 * @since 0.97.0.24
 	 */
 	private boolean checkForLegacyDatabaseConfig() {
-		File file = getDataFolder().toPath().resolve("settings").resolve("config.yml").toFile();
+		Path configYMLPath = getDataFolder().toPath().resolve("settings").resolve("config.yml");
 		// Bail if the config doesn't exist at all yet.
-		if (!file.exists())
+		if (Files.exists(configYMLPath))
```

Comment:
This looks like backwards logic. We're supposed to bail on converting from a config that doesn't exist yet. This code is run before the config will be generated for the first time on a fresh install.

Reply:
Yeah good catch, fixed.